postgres-appliance/scripts/configure_spilo.py

```diff
@@ -127,7 +127,7 @@ def deep_update(a, b):
                 a[key] = b[key]
         return a
     if type(a) is list and type(b) is list:
-        return a + [i for i in b if i not in a]
+        return a
```

Comment:
The function description needs to be adjusted accordignly

Reply:
I agree